dsync/send.go

```diff
@@ -48,6 +49,27 @@ type Send struct {
 }
 
 // Do executes the send, blocking until complete
+// First we send a manifest to the remote node
```

Comment:
Similar the `fetch.Do` comment, describing the internals of the sync process doesn't help the consumer of this package use `Send`. These descriptions of processes would be helpful of they were generalized & worked into the main package comment that describes the goals of dsync